script.js

```diff
@@ -106,4 +106,22 @@ document.querySelector('#container-book-list').addEventListener('click', (e) =>
   const id = parseInt(idString, 10);
   store.removeBook(id);
   UI.hideOrRemoveFieldet();
+});
+
+document.querySelector('#book-list-menu').addEventListener('click', () => {
+  document.querySelector('.books-container').classList.remove('hide');
+  document.querySelector('.form').classList.add('hide');
+  document.querySelector('.section-contact-info').classList.add('hide');
+});
+
+document.querySelector('#add-new-book-menu').addEventListener('click', () => {
+  document.querySelector('.form').classList.remove('hide');
+  document.querySelector('.books-container').classList.add('hide');
+  document.querySelector('.section-contact-info').classList.add('hide');
+});
```

Comment:
[OPTIONAL] When I add a bok I still remain on the add book page. I think it will be a better UI if I am automatically rerouted to the list of books page. This assures me that the book was correctly added without the need to check mysel.